gutenberg.php

```diff
@@ -70,7 +70,7 @@ function gutenberg_menu() {
 	if ( gutenberg_is_fse_theme() ) {
 		add_menu_page(
 			__( 'Site Editor (beta)', 'gutenberg' ),
-			__( 'Site Editor (beta)', 'gutenberg' ),
+			__( 'Site Editor <span class="awaiting-mod">beta</span>', 'gutenberg' ),
```

Comment:
Would it be better for translators if the `<span>` is inside a `sprintf`? 🤔 
```suggestion
			sprintf(
				/* translators: %s: "beta" label. */
				__( 'Site Editor %s', 'gutenberg' ),
				'<span class="awaiting-mod">' . __( 'beta', 'gutenberg' ) . '</span>'
			),
```

Reply:
Makes sense, thanks.